docs/operator-guides/reset.md

```diff
@@ -2,29 +2,29 @@
 products: all
 ---
 
-# Resetting your data
+# Clearing your data
 
-Resetting your data allows you to drop all previously synced data so that any ensuing sync can start syncing fresh. This is useful if you don't require the data replicated to your destination to be saved permanently or are just testing Airbyte.
+Clearing your data allows you to drop all previously synced data so that any ensuing sync can start syncing fresh. This is useful if you don't require the data replicated to your destination to be saved permanently or are just testing Airbyte.
 
-Airbyte allows you to reset all streams in the connection, some, or only a single stream (when the connector support per-stream operations).
+Airbyte allows you to clear all streams in the connection, some, or only a single stream.
 
-A sync will automatically start after a completed reset, which commonly backfills all historical data.
+In order to backfill all historical data, a sync should be initiated after your clear succeeds.
 
-## Performing a Reset
-To perform a full reset that resets all your streams, select `Reset your data` in the UI on a connection's status or job history tabs by selecting the three grey dots next to "Sync now". 
+## Clearing your data
```

Comment:
nit: above, it mentions the ability to clear "some" streams... is it worth mentioning somewhere that this is only supported via the API?

Reply:
Added that note!